Parsing_onliner_project/categories.py

```diff
@@ -0,0 +1,75 @@
+"""Storage module for class OnlinerArticle, OnlinerCategory and MainOnlinerPageLinks"""
```

Comment:
Folders could not be named with Capital letters.
It should be named as "models" and file itself should be "onliner_html_models.py"